src/utils/helpers/review/getReviewStartLabel.ts

```diff
@@ -1,8 +1,12 @@
 import strings from '../../constants'
 import { REVIEW_STATUS } from '../../data/reviewStatus'
 
+/**
+ * @function getReviewStartLabel
+ * Get label string of option to display on Review start page
+ * @param status Current status of review
+ */
 const getReviewStartLabel = (status: string) => {
-  // TODO: Not use strings here - use the type
   switch (status as REVIEW_STATUS) {
     case REVIEW_STATUS.DRAFT:
```

Comment:
Is there any way to somehow unify the `ReviewStatus` type with the `REVIEW_STATUS` enum? And then have `(status: ReviewStatus) => ...` in the parameters rather than `string`?

Reply:
Well, that's problem that I would like to see how to resolve. The problem is that when we set the type to be `ReviewStatus` the status is something as `"Draft"`, `"Changes Required"` and the `ReviewStatus.Draft` (a Graphql enum) maps to DRAFT, so the comparison doesn't work. That's why I'm comparing with this built-in type REVIEW_STATUS. I have changed the received type to ReviewStatus in 78017ea

Reply:
Fixed in PR #327